test/algorithm/partial_sort_copy.cpp

```diff
@@ -139,11 +140,13 @@ TEST_CASE("alg.partial_sort_copy")
         for (int i = 0; i < N; ++i)
             input[i].i = i;
         std::shuffle(input, input+N, gen);
-        U* r = stl2::partial_sort_copy(input, output, std::less<int>(), &S::i, &U::i);
-        U* e = output + std::min(N, M);
-        CHECK(r == e);
+	auto r = stl2::partial_sort_copy(input, output, std::less<int>(), &S::i, &U::i);
+        S* e_in = input + std::min(N, M);
+        U* e_out = output + std::min(N, M);
+        //CHECK(r.in == e_in); // Fails!
```

Comment:
Just calling attention to this line. I tried it, but the assertion failed.

Reply:
What is the expected value of `r.in` here? Is this a bug in `partial_sort_copy`?

Reply:
I think I messed up the return value of this and the new `subrange<I>`-returning algorithms. Can you just confirm that I'm reading the draft correctly?